src/plays/basic-calculator/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Basic Calculator 
+
+This is a Simple calculator where one can do addition, subtraction, multiplication, division. All clear and delete facility is also there. No external library is used to make this play
+
+## Play Demographic
+
+- Language: js
+- Level: Beginner
+
+## Creator Information
+
+- User: bahnisikhadhar
+- Gihub Link: https://github.com/bahnisikhadhar
+- Blog: 
+- Video: 
```

Comment:
Kindly record a video about your demo on stack stream.

Reply:
ok will do that